src/realm_dart.cpp

```diff
@@ -99,21 +99,18 @@ RLM_API void realm_dart_delete_persistent_handle(void* handle) {
     Dart_DeletePersistentHandle_DL(persistentHandle);
 }
 
-RLM_API realm_dart_userdata_async_t realm_dart_userdata_async_new(Dart_Handle handle, void* callback, realm_scheduler_t* scheduler)
-{
+RLM_API realm_dart_userdata_async_t realm_dart_userdata_async_new(Dart_Handle handle, void* callback, realm_scheduler_t* scheduler) {
```

Comment:
I wonder if we should start using `clang-format`?

Reply:
lets discuss. seems like editor config is not working or something